TestProjects/Forms/Example002CSharp/Example/Example.csproj

```diff
@@ -8,10 +8,11 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Example</RootNamespace>
     <AssemblyName>Example</AssemblyName>
-    <TargetFrameworkProfile>Profile259</TargetFrameworkProfile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkProfile>Profile24</TargetFrameworkProfile>
```

Comment:
I don't think it is good to change this. Normally we use Profile7 with .NET 4.5. This would be fixed when you merge upstream, where this sample is removed.